src/com/codeenginestudio/bookManagement/manage/ManageBook.java

```diff
@@ -31,14 +31,14 @@ public void deleteBook(int bookId) {
 		}
 	}
 
-	public void editBook(int bookId, String bookName, String bookAuthor) {
+	public static void editBook(int bookId, String bookName, String bookAuthor) {
 
 		Book book = getOneBook(bookId);
 		book.setBookName(bookName);
 		book.setBookAuthor(bookAuthor);
 	}
 
-	public Book getOneBook (int bookId) {
+	public static Book getOneBook (int bookId) {
```

Comment:
@HuongNguyenThiThu @thuycao625 The `get` method will be throw exception if it doesnot exist. The `fetch` method will return null if it doesnot exist.